scripts/eqtl_hail_batch/launch_eqtl_spearman.py

```diff
@@ -783,6 +784,9 @@ def spearman_correlation(df):
     # for front-end analysis
     spearman_df = t.to_pandas()
     spearman_df['round'] = '1'
+    # change bp and round into int values
+    spearman_df['round'].astype(str).astype(int)
+    spearman_df['bp'].astype(str).astype(int)
```

Comment:
Do you need two `astype` calls here (and below) or would `spearman_df['round'].astype(int)` work too?

Reply:
I think you're right, it would work. I didn't take notes for this, unfortunately, but I remember some part breaking when I had `round` as an int, so I was just being precautious and doing it after since it didn't fail locally. Should be the same though (and I'll change this).

Reply:
I think you're right! I'll remove the redundant `astype(str)` bit